### ISSUE TITLE
Cannot find module '../creds.json'

### ISSUE THREAD

**Issue by ekkis** [aemc/bitcoin-testnet3#6]
the documentation states:

> Copy & paste the files creds.json and keys.json provided via email into the project root folder.

but gives the user no idea how to reques that such files be sent.  it would be useful if the project included these files so the user can modify as needed.  as it is, I can't run the app

**Comment by aemc** [comment]
I agree the documentation can be better. The secrets come from https://www.blockcypher.com/

**Comment by ekkis** [comment]
oh.  I haven't been able to log into that bloody site.  grr... that means I can't use this module
